types/twig/index.d.ts

```diff
@@ -54,7 +54,7 @@ export interface RenderOptions {
 }
 
 export function twig(params: Parameters): Template;
-export function extendFilter(name: string, definition: (left: any, ...params: any[]) => string): void;
+export function extendFilter(name: string, definition: (left: any, params: any[] | false) => string): void;
```

Comment:
Following the wiki, the `params` argument is optional (https://github.com/twigjs/twig.js/wiki/Extending-twig.js#filters).

You should only put the argument optional.

Reply:
@emmanuelgautier the documentation is showing a function that doesn't define the last argument, which is not the same as the argument being optional; in JS it's valid for functions to not define the last argument(s) if they're being used.

In my testing, Twig _always_ passes a value for `params`, so it shouldn't be marked as optional here.